src/version.ts

```diff
@@ -201,3 +205,16 @@ export function getLatestMatching(
   }
   return getLatest(matching_versions);
 }
+
+export function getArchCandidates(use_32bits: boolean): string[] {
+  if (process.platform === 'darwin') {
+    return ['universal', 'x86_64'];
+  }
+  if (process.arch === 'x64') {
+    return use_32bits ? ['x86'] : ['x86_64', 'x86'];
+  } else if (process.arch === 'arm64') {
+    return use_32bits ? ['arm'] : ['aarch64', 'arm'];
+  } else {
```

Comment:
Do we have tests that more or less cover all these branches? If not is it straightforward to add them?

Reply:
The tests cover some but I don't think all of these cases. I'll give it an audit

Reply:
It looks like cmake doesn't actually release 32-bit `arm` packages: https://github.com/Kitware/CMake/releases/tag/v4.2.1

So I think it would make sense to ignore `use_32bits` when `arch === 'arm64'` and just always return `aarch64`. Will update